2021-07-01 13:13:32,171 INFO    MainThread:14764 [wandb_setup.py:_flush():69] setting env: {}
2021-07-01 13:13:32,171 INFO    MainThread:14764 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-01 13:13:32,172 INFO    MainThread:14764 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-01 13:13:32,172 INFO    MainThread:14764 [wandb_init.py:_log_setup():337] Logging user logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\solution-guidance\wandb\run-20210701_131332-lzwtwrj2\logs\debug.log
2021-07-01 13:13:32,173 INFO    MainThread:14764 [wandb_init.py:_log_setup():338] Logging internal logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\solution-guidance\wandb\run-20210701_131332-lzwtwrj2\logs\debug-internal.log
2021-07-01 13:13:32,175 INFO    MainThread:14764 [wandb_init.py:init():370] calling init triggers
2021-07-01 13:13:32,175 INFO    MainThread:14764 [wandb_init.py:init():377] wandb.init called with sweep_config: {}
config: {}
2021-07-01 13:13:32,175 INFO    MainThread:14764 [wandb_init.py:init():392] re-initializing run, found existing run on stack: p24gwlm8
2021-07-01 13:13:32,177 INFO    MainThread:14764 [wandb_run.py:finish():1294] finishing run abhishekdiphu/IBM-Project/p24gwlm8
2021-07-01 13:13:32,181 INFO    MainThread:14764 [jupyter.py:save_history():339] not saving jupyter history
2021-07-01 13:13:32,181 INFO    MainThread:14764 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-07-01 13:13:32,181 INFO    MainThread:14764 [wandb_init.py:_jupyter_teardown():269] cleaning up jupyter logic
2021-07-01 13:13:32,181 INFO    MainThread:14764 [wandb_run.py:_atexit_cleanup():1594] got exitcode: 0
2021-07-01 13:13:32,182 INFO    MainThread:14764 [wandb_run.py:_restore():1566] restore
2021-07-01 13:13:32,710 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 1
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 103476
  total_bytes: 103476
}

2021-07-01 13:13:33,099 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 1
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 103476
  total_bytes: 103476
}

2021-07-01 13:13:33,506 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 103476
  total_bytes: 107543
}

2021-07-01 13:13:33,890 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 103476
  total_bytes: 107543
}

2021-07-01 13:13:33,992 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107543
  total_bytes: 107543
}

2021-07-01 13:13:34,094 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107543
  total_bytes: 107543
}

2021-07-01 13:13:34,196 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107543
  total_bytes: 107543
}

2021-07-01 13:13:34,297 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107543
  total_bytes: 107543
}

2021-07-01 13:13:34,399 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107543
  total_bytes: 107543
}

2021-07-01 13:13:34,503 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107543
  total_bytes: 107543
}

2021-07-01 13:13:34,605 INFO    MainThread:14764 [wandb_run.py:_wait_for_finish():1716] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107543
  total_bytes: 107543
}

2021-07-01 13:13:35,766 INFO    MainThread:14764 [wandb_run.py:_show_summary():1871] rendering summary
2021-07-01 13:13:35,769 INFO    MainThread:14764 [wandb_run.py:_show_history():1909] rendering history
2021-07-01 13:13:35,772 INFO    MainThread:14764 [wandb_run.py:_show_files():1938] logging synced files
2021-07-01 13:13:35,788 INFO    MainThread:14764 [wandb_init.py:init():419] starting backend
2021-07-01 13:13:35,788 INFO    MainThread:14764 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=spawn, using: spawn
2021-07-01 13:13:35,797 INFO    MainThread:14764 [backend.py:ensure_launched():135] starting backend process...
2021-07-01 13:13:35,993 INFO    MainThread:14764 [backend.py:ensure_launched():140] started backend process with pid: 19392
2021-07-01 13:13:35,994 INFO    MainThread:14764 [wandb_init.py:init():424] backend started and connected
2021-07-01 13:13:36,019 INFO    MainThread:14764 [wandb_run.py:_label_probe_notebook():845] probe notebook
2021-07-01 13:13:36,019 INFO    MainThread:14764 [wandb_run.py:_label_probe_notebook():852] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-07-01 13:13:36,020 INFO    MainThread:14764 [wandb_init.py:init():472] updated telemetry
2021-07-01 13:13:36,026 INFO    MainThread:14764 [wandb_init.py:init():491] communicating current version
2021-07-01 13:13:39,445 INFO    MainThread:14764 [wandb_init.py:init():496] got version response upgrade_message: "wandb version 0.10.33 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-07-01 13:13:39,446 INFO    MainThread:14764 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-07-01 13:13:39,952 INFO    MainThread:14764 [wandb_init.py:init():529] starting run threads in backend
2021-07-01 13:13:41,058 INFO    MainThread:14764 [wandb_run.py:_console_start():1624] atexit reg
2021-07-01 13:13:41,060 INFO    MainThread:14764 [wandb_run.py:_redirect():1498] redirect: SettingsConsole.WRAP
2021-07-01 13:13:41,060 INFO    MainThread:14764 [wandb_run.py:_redirect():1535] Wrapping output streams.
2021-07-01 13:13:41,065 INFO    MainThread:14764 [wandb_run.py:_redirect():1559] Redirects installed.
2021-07-01 13:13:41,065 INFO    MainThread:14764 [wandb_init.py:init():554] run started, returning control to user process
2021-07-01 13:13:47,419 INFO    MainThread:14764 [wandb_run.py:_config_callback():873] config_cb ('_wandb', 'viz', 'outlier_candidates') {'id': 'wandb/outliers/v1', 'historyFieldSettings': {'key': 'outlier_candidates', 'x-axis': '_step'}} None
2021-07-01 13:14:41,254 INFO    MainThread:14764 [wandb_run.py:_config_callback():873] config_cb ('_wandb', 'viz', 'learning_curve') {'id': 'wandb/learning_curve/v1', 'historyFieldSettings': {'key': 'learning_curve', 'x-axis': '_step'}} None
2021-07-01 13:14:47,497 INFO    MainThread:14764 [wandb_run.py:_config_callback():873] config_cb ('_wandb', 'viz', 'residuals') {'id': 'wandb/residuals_plot/v1', 'historyFieldSettings': {'key': 'residuals', 'x-axis': '_step'}} None
2021-07-01 13:15:00,962 INFO    MainThread:14764 [wandb_setup.py:_flush():69] setting env: {}
2021-07-01 13:15:00,963 INFO    MainThread:14764 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-01 13:15:00,963 INFO    MainThread:14764 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-01 13:15:00,963 INFO    MainThread:14764 [wandb_init.py:_log_setup():337] Logging user logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\solution-guidance\wandb\run-20210701_131500-195j1wf8\logs\debug.log
2021-07-01 13:15:00,964 INFO    MainThread:14764 [wandb_init.py:_log_setup():338] Logging internal logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\solution-guidance\wandb\run-20210701_131500-195j1wf8\logs\debug-internal.log
2021-07-01 13:15:00,965 INFO    MainThread:14764 [wandb_init.py:_jupyter_setup():288] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001F18D635B00>
2021-07-01 13:15:00,966 INFO    MainThread:14764 [wandb_init.py:init():370] calling init triggers
2021-07-01 13:15:00,966 INFO    MainThread:14764 [wandb_init.py:init():377] wandb.init called with sweep_config: {}
config: {}
2021-07-01 13:15:00,966 INFO    MainThread:14764 [wandb_init.py:init():392] re-initializing run, found existing run on stack: lzwtwrj2
2021-07-01 13:15:00,969 INFO    MainThread:14764 [wandb_run.py:finish():1294] finishing run abhishekdiphu/IBM-Project/lzwtwrj2
